runtimes/peregrine/src/lib.rs

```diff
@@ -685,6 +687,111 @@ impl pallet_utility::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+// Start of proxy pallet configuration
+
+parameter_types! {
+	// One storage item; key size 32, value size 8; .
+	pub const ProxyDepositBase: Balance = KILT;
+	// Additional storage item size of 33 bytes.
+	pub const ProxyDepositFactor: Balance = KILT;
+	pub const MaxProxies: u16 = 32;
+	pub const AnnouncementDepositBase: Balance = KILT;
+	pub const AnnouncementDepositFactor: Balance = KILT;
+	pub const MaxPending: u16 = 32;
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, MaxEncodedLen, scale_info::TypeInfo,
+)]
+pub enum ProxyType {
+	Any,
+	NonTransfer,
+	Governance,
+	Staking,
+	CancelProxy,
+	Ctype,
```

Comment:
Ctype, attestation, delegation and DIDs work at the DID level, and they are already proxied via `Did::submit_did_call()`, so I am not sure having a proxy for those would make sense here. Especially since the proxy is not generic over the definition of an account but it takes whatever is set as `frame_system::AccountId`, which in our case are KILT accounts and not DIDs.

Reply:
removed them, very good point, it even makes it simpler :man_shrugging: